bullet_train-themes-light/app/assets/stylesheets/light/electron.css

```diff
@@ -6,8 +6,8 @@
   a {
     outline-color: var(--tw-ring-color);
     /* TODO: Will --tw-ring-opacity still be available in tailwindcss v4? */
-    --tw-ring-color: rgba(3, 98, 198, var(--tw-ring-opacity));
     --tw-ring-opacity: 1;
+    --tw-ring-color: rgba(3, 98, 198, var(--tw-ring-opacity));
```

Comment:
If we declare `--tw-ring-opacity` before trying to reference it with `var` then the complaint goes away.